tikv-client/src/test/java/com/pingcap/tikv/txn/LockResolverRCTest.java

```diff
@@ -67,38 +68,39 @@ public void RCTest() {
       skipTest();
       return;
     }
-    TiTimestamp startTs = pdClient.getTimestamp(backOffer);
-    TiTimestamp endTs = pdClient.getTimestamp(backOffer);
+    TiTimestamp startTs = session.getTimestamp();
+    TiTimestamp endTs = session.getTimestamp();
 
     // Put <a, a> into kv
     putKV("a", "a", startTs.getVersion(), endTs.getVersion());
 
-    startTs = pdClient.getTimestamp(backOffer);
-    endTs = pdClient.getTimestamp(backOffer);
+    startTs = session.getTimestamp();
+    endTs = session.getTimestamp();
 
     // Prewrite <a, aa> as primary without committing it
     assertTrue(lockKey("a", "aa", "a", "aa", false, startTs.getVersion(), endTs.getVersion()));
 
     TiRegion tiRegion = session.getRegionManager().getRegionByKey(ByteString.copyFromUtf8("a"));
     RegionStoreClient client = builder.build(tiRegion);
-    // In RC mode, lock will not be read. <a, a> is retrieved.
-    ByteString v =
-        client.get(
-            backOffer, ByteString.copyFromUtf8("a"), pdClient.getTimestamp(backOffer).getVersion());
-    assertEquals(v.toStringUtf8(), "a");
+
+    {
```

Comment:
why do we have `{` here?

Reply:
to indicate that it is an independent logic block